docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -2477,6 +2477,28 @@ const markAllInboxNotificationsAsRead = useMarkAllInboxNotificationsAsRead();
 markAllInboxNotificationsAsRead();
 ```
 
+### useDeleteInboxNotification [@badge=LiveblocksProvider]
+
+Returns a function that deletes an inbox notification.
```

Comment:
```suggestion
Returns a function that deletes an inbox notification for the current user.
```

This only works for the current user's notifications, right? Maybe should clarify that

Reply:
Yep technically inbox notifications are user-based so you will always only see yours but I'll clarify 👍